Parameters.ps1

```diff
@@ -0,0 +1,10 @@
+   # Variables for the registration of the AAD application for the Web API Service
+    $serviceAadAppName = "TodoListService"
+    $serviceHomePage = "https://localhost:44321"
+    $serviceAppIdURI = "https://$tenantName/$serviceAadAppName"
+
+    # Variables for the registration of the AAD application for the Daemon app
+    $daemonAadAppName = "TodoListDaemonWithCert"
+    $daemonHomePage = "http://TodoListDaemonWithCert"
```

Comment:
The CI build confirmed that the change does not break the sample. 
I love the CI build 👍